<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>RecoverWidget</class>
 <widget class="QWidget" name="RecoverWidget">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>713</width>
    <height>82</height>
   </rect>
  </property>
  <layout class="QGridLayout" name="gridLayout">
   <item row="1" column="0" rowspan="2">
    <widget class="QLabel" name="lblIcon">
     <property name="minimumSize">
      <size>
       <width>15</width>
       <height>0</height>
      </size>
     </property>
    </widget>
   </item>
   <item row="1" column="1">
    <widget class="QLabel" name="lblSwap">
     <property name="text">
      <string>There has been a swap file. Do you want to recover it?</string>
     </property>
    </widget>
   </item>
   <item row="2" column="1">
    <layout class="QHBoxLayout" name="horizontalLayout">
     <item>
      <widget class="QCheckBox" name="chkIgnoreWhiteSpaces">
       <property name="sizePolicy">
        <sizepolicy hsizetype="Expanding" vsizetype="Fixed">
         <horstretch>0</horstretch>
         <verstretch>0</verstretch>
        </sizepolicy>
       </property>
       <property name="text">
        <string>Ignore white space changes</string>
       </property>
      </widget>
     </item>
     <item>
      <widget class="KPushButton" name="btnDiff">
       <property name="text">
        <string>View Differences</string>
       </property>
      </widget>
     </item>
     <item>
      <spacer name="horizontalSpacer">
       <property name="orientation">
        <enum>Qt::Horizontal</enum>
       </property>
       <property name="sizeHint" stdset="0">
        <size>
         <width>98</width>
         <height>20</height>
        </size>
       </property>
      </spacer>
     </item>
     <item>
      <widget class="KPushButton" name="btnRecover">
       <property name="text">
        <string>Recover</string>
       </property>
      </widget>
     </item>
     <item>
      <widget class="KPushButton" name="btnDiscard">
       <property name="text">
        <string>Discard</string>
       </property>
      </widget>
     </item>
    </layout>
   </item>
  </layout>
 </widget>
 <customwidgets>
  <customwidget>
   <class>KPushButton</class>
   <extends>QPushButton</extends>
   <header>kpushbutton.h</header>
  </customwidget>
 </customwidgets>
 <tabstops>
  <tabstop>chkIgnoreWhiteSpaces</tabstop>
  <tabstop>btnDiff</tabstop>
  <tabstop>btnRecover</tabstop>
  <tabstop>btnDiscard</tabstop>
 </tabstops>
 <resources/>
 <connections/>
</ui>
